### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by AswathyVNair**
NAVIGATION
navigation

preview:https://deploy-preview-8--asthra-ui.netlify.app/navigation/navigation.html

### CODE REVIEW

**Review by itsmebipul00 — APPROVED** [review]
Looks nice, clean code good to merge

File: navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.btn-nav{
+    display: flex;
+    width: 20%;
+    justify-content: space-between;
+}
+
+.btn-nav button{
+    font-size: 1.1rem;
+    border: none;
+    padding: 0.5rem 1rem 0.5rem 1rem;
```
Comment (zoef-yt) [inline navigation/navigation.css]: This can be written as `padding:0.5rem 1rem`

**Review by zoef-yt — APPROVED** [review]
The code looks good. Do the mentioned changes else the code. Good to merge.

File: navigation/navigation.css (hunk 1 above)
Comment (AswathyVNair) [reply navigation/navigation.css]: Thank you for letting me know this. Will change that.

**Review by Muzammil-khan710 — APPROVED** [review]
Good to go !

**Pull request merged**